### ISSUE TITLE
Unexpected assertion error with `satisfiesContract()` when impl has more operations than design apiDoc under same path

### ISSUE THREAD

**Issue by atsu85** [RobWin/assertj-swagger#26]
# Expected behavior
When design api-doc includes less operations from the same path compared to implementation api-doc, then 
```java
SwaggerAssertions.assertThat(implApiDoc)
                .satisfiesContract(designApiDoc);
```
shouldn't fail...

# Actual behaviour
...but actually currently [the test](https://github.com/RobWin/assertj-swagger/pull/27/commits/f3724babc0b001ec8a74a2d521aace35f7af03bc) fails with an error having output similar to following:
```
org.assertj.core.api.SoftAssertionError: 
The following 2 assertions failed:
1) [Checking 'DELETE' operation of path '/pets/{petId}'] expected:<null> but was:<io.swagger.models.Operation@79d154eb[deletePet]>
at ConsumerDrivenValidator.validateOperation(ConsumerDrivenValidator.java:259) expected:<null> but was:<io.swagger.models.Operation@79d154eb[deletePet]>
2) [Checking 'POST' operation of path '/pets/{petId}'] expected:<null> but was:<io.swagger.models.Operation@53f708e3[updatePetWithForm]>
at ConsumerDrivenValidator.validateOperation(ConsumerDrivenValidator.java:259) expected:<null> but was:<io.swagger.models.Operation@53f708e3[updatePetWithForm]>


	at org.assertj.core.api.SoftAssertions.assertAll(SoftAssertions.java:133)
	at io.github.robwin.swagger.test.ConsumerDrivenValidator.validateSwagger(ConsumerDrivenValidator.java:75)
	at io.github.robwin.swagger.test.SwaggerAssert.satisfiesContract(SwaggerAssert.java:97)
	at io.github.robwin.swagger.test.SwaggerAssert.satisfiesContract(SwaggerAssert.java:109)
	at io.github.robwin.swagger.SwaggerConsumerDrivenAssertTest.shouldntFailWhenImplHasMoreOperationsOfSamePath(SwaggerConsumerDrivenAssertTest.java:149)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
```

**Comment by RobWin** [comment]
Thx for creating an issue.

**Comment by atsu85** [comment]
@RobWin , could you take a look at the tiny [PR](https://github.com/RobWin/assertj-swagger/pull/27) that fixes this issue?

**Comment by RobWin** [comment]
Yes but not today

**Comment by atsu85** [comment]
ok, i discovered this issue when preparing test & fix for issue #28
